src/docdb/registerDocDBCommands.ts

```diff
@@ -3,6 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { doubleClickDebounceDelay } from "src/constants";
```

Comment:
These have to be relative paths like `../../constants.ts` (I'm guessing that's why the build is failing)

VS Code just recently started autocompleting to a path like this. It's super annoying and I want to try to fix it as a part of some engineering work eventually

Reply:
@EricJizbaMSFT This shouldn't be entered as an issue against vscode?

Reply:
It looks like it's already entered: 
https://github.com/Microsoft/vscode/issues/59506

The fix is changing the setting `typescript.preferences.importModuleSpecifier` from auto to relative.

Reply:
No I want to get these types of imports working:
```
import { doubleClickDebounceDelay } from "src/constants";
```

I think they look much cleaner and are easier to refactor/change than this:
```
import { doubleClickDebounceDelay } from "../../constants.ts"
```

This is all related to the `baseUrl` setting I added to our tsconfig.json back in August. I did it for an unrelated reason (see [this PR](https://github.com/Microsoft/vscode-cosmosdb/pull/801) for why I originally added it), but it's main point is to support the imports like above. I think VS Code just recently added support for the `baseUrl` property as a part of it's autocomplete which is why we're seeing this now.

Reply:
But I kind of want to wait to do any 'import' or 'module' stuff until after you do webpack :)

Reply:
@PrashanthCorp nice find! Definitely worth changing that in the repo's `.vscode/settings.json` (but I still want to change our imports eventually as described above)